opa/README.md

```diff
@@ -16,4 +16,14 @@ oc apply -f gatekeeper.yaml
 # Install policy
 oc apply -f policy-resources.yaml
 
-```
\ No newline at end of file
+```
+
+## NOTE: for every Advanced Cluster Manager spoke cluster participating in the Open Policy Agent & Gatekeeper integration run the following commands:
+```bash
+# Install Gatekeeper
+oc apply -f gatekeeper.yaml
+
+# Add priviledged securitu context constraints for the gatekeeper-admin service account  
+oc adm policy add-scc-to-user privileged -z gatekeeper-admin -n gatekeeper-system
```

Comment:
I don't believe this is needed - see my comment in WhatsApp

Reply:
10-4